### PR TITLE
73 responsive whattodo featurework

### PR DESCRIPTION
**Pull request by HarryuCATGREY**
### Descriptions

- Link to UI design (if possible): https://www.figma.com/file/q2ZHM8s0LcNxz3otGFLDzO/Website?node-id=0%3A1&t=U1p59QlVKt5XJoh5-0

### Provide screenshots or videos that the function is working.

##744px width
image
image
image

##390px width
image
image
image

### Provide screenshots of no warning, or error in the console.
image

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

**Review by huchshm — APPROVED** [review]
LGTM

File: components/general/FeaturedWork.tsx [hunk 1]
```diff
@@ -33,12 +33,12 @@ export default function FeaturedWork() {
     },
   ]);
 
+  let [titleName, setTitleName] = useState("global__heading-h1");
+  let [brExist, setBrExist] = useState(true);
```
Comment (Charles-Xing) [inline components/general/FeaturedWork.tsx:37]: good idea, maybe using the flexbox method would be simpler?

**Comment by Charles-Xing** [comment]
It looks like there are still some issues with responsiveness in the mobile mode.
Q1: the style of the big title, fontsize?
Q2: the responsive of the picture.Because the height is fixed, when the width is reduced, the content on the left and right sides of the image will disappear instead of being directly scaled by a percentage.

File: components/general/FeaturedWork.tsx (hunk 1 above)
Comment (HarryuCATGREY) [reply components/general/FeaturedWork.tsx:37]: Actually, they are the useless codes, I want to discuss these in later stand up meeting

**Comment by jiayuu-u** [comment]
image
image

Figma shows there has padding between the icon and the text.

**Comment by jiayuu-u** [comment]
image
It looks don't have the same padding.

**Comment by HarryuCATGREY** [comment]
> It looks like there are still some issues with responsiveness in the mobile mode. Q1: the style of the big title, fontsize? Q2: the responsive of the picture.Because the height is fixed, when the width is reduced, the content on the left and right sides of the image will disappear instead of being directly scaled by a percentage.

Q1 is solved Q2 can be solved, but I think it is better to follow the prototype.

**Comment by HarryuCATGREY** [comment]
> 

The prob is fixed now.

**Comment by HarryuCATGREY** [comment]
> image It looks don't have the same padding.

The prob is fixed now.

**Comment by Charles-Xing** [comment]
> 
OK

**Review by yunshen513 — APPROVED** [review]

**Pull request merged**